Makefile

```diff
@@ -128,6 +128,17 @@ check-yml: $(TOOLING) ## Check the YAML files
 		-c .yamllint.yml \
 		.
 
+.PHONY: reproducible-build
+reproducible-build: build ## Check if the container is reproducible
```

Comment:
This command should be added to the [contributing guidelines](https://github.com/ericcornelissen/js-regex-security-scanner/blob/dbe4b993186f28cdbf35f6a4947744fbc5470797/CONTRIBUTING.md?plain=1#L115-L127) too.

Reply:
Resolved with 79925dbdb89e7fde887a9646cb2d2d0312e6106e